ambry-server/src/integration-test/java/com/github/ambry/server/ServerTestUtil.java

```diff
@@ -479,6 +484,72 @@ static void endToEndTest(Port targetPort, String routerDatacenter, MockCluster c
       releaseNettyBufUnderneathStream(stream);
       assertEquals(ServerErrorCode.Bad_Request, adminResponseWithContent.getError());
 
+      //
+      // Start the test of ForceDeleteAdminRequest
+      //
+      // Force delete on a blob which doesn't exist.
+      short lifeVersion = 1;
+      long nowTime = System.currentTimeMillis();
+      ForceDeleteAdminRequest forceDeleteAdminRequest = new ForceDeleteAdminRequest(blobId6, lifeVersion,
+          new AdminRequest(AdminRequestOrResponseType.ForceDelete, partitionIds.get(0), 1, "clientid2"));
+      stream = channel.sendAndReceive(forceDeleteAdminRequest).getInputStream();
+      AdminResponse adminResponse = AdminResponse.readFrom(stream);
+      releaseNettyBufUnderneathStream(stream);
+      assertEquals(ServerErrorCode.No_Error, adminResponse.getError());
+      // Use BlobIndexAdminRequest to verify we have one message info.
+      blobIndexAdminRequest = new BlobIndexAdminRequest(blobId6,
+          new AdminRequest(AdminRequestOrResponseType.BlobIndex, partitionIds.get(0), 1, "clientid2"));
+      stream = channel.sendAndReceive(blobIndexAdminRequest).getInputStream();
+      adminResponseWithContent = AdminResponseWithContent.readFrom(stream);
+      releaseNettyBufUnderneathStream(stream);
+      assertEquals(new String(adminResponseWithContent.getContent()), ServerErrorCode.No_Error,
+          adminResponseWithContent.getError());
+      jsonBytes = adminResponseWithContent.getContent();
+      objectMapper = new ObjectMapper();
+      StoreKeyJacksonConfig.setupObjectMapper(objectMapper, new BlobIdFactory(clusterMap));
```

Comment:
same here

Reply:
For the following two lines,
objectMapper = new ObjectMapper();
StoreKeyJacksonConfig.setupObjectMapper(objectMapper, new BlobIdFactory(clusterMap));

I thought I didn't need these two lines either, but I have to add them back, missing one will cause the exception somehow. 

Cannot construct instance of `com.github.ambry.store.StoreKey` (no Creators, like default constructor, exist): abstract types either need to be mapped to concrete types, have custom deserializer, or contain additional type information
```suggestion
      StoreKeyJacksonConfig.setupObjectMapper(objectMapper, new BlobIdFactory(clusterMap));
```

Reply:
Looks like it's related to the following line.

> https://jarvis.corp.linkedin.com/codesearch/result/?name=ServerTestUtil.java&path=ambry%2Fambry-server%2Fsrc%2Fintegration-test%2Fjava%2Fcom%2Fgithub%2Fambry%2Fserver&reponame=linkedin%2Fambry#458
 
After I removed the above line, I don't need reinitialize it any more.